docs/snippets/common/storybook-preview-enable-toc.js.mdx

```diff
@@ -0,0 +1,11 @@
+```js
+// .storybook/preview.js
+
+export default {
+  parameters: {
+    docs: {
+      toc: true, // 👈 Enables the table of contents
```

Comment:
Are you sure this typechecks?

Reply:
Was able to get it running without any issues based on this [reproduction](https://github.com/jonniebigodes/sample-toc-options-sb-ts).